app/assets/stylesheets/components/_cards.scss

```diff
@@ -12,6 +12,7 @@
 // Shared cards
 
 .card-container {
+  height: 22rem;
```

Comment:
I already added a fixed height to card body itself in the last merge which also resolved the problem.
.card-body-bottom {
 height: 35px;
}

 I think both does the same thing. Let's keep your height setting too and see how it looks.